### DISCUSSION TITLE
How to know my total plots from multiple farmer machine?

### DISCUSSION THREAD

**Discussion by michealChin** [Chia-Network/chia-blockchain#5067] [Support]
Hi,

I am new to Chia blockchain, I would like to know how to get my total plots from multiple farmer machine that connected to the blockchain with the same farmer key? Thanks!

**Comment by khituras** [comment]
I am wondering similar things. I now have a main machine and two harvester-only machine, set up according to https://github.com/Chia-Network/chia-blockchain/wiki/Farming-on-many-machines. Now I wonder if the main machine can give me aggregated information across all harvesters: How many plots are being harvested in total? What is the time to win when taking into account all harvesters?

**Comment by lvcivs** [comment]
I believe this is being worked on and will likely be in one of the next releases.

**Comment by BrandtH22** [comment]
If you are still experiencing the issues it is recommended to upgrade to the latest version of chia. Please download it from here: https://www.chia.net/downloads/

Since this issue has been open for a number of years without additional comments, we will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions, comments, or if the issue persists after an update.

```
Please note that upgrading major versions of chia generally requires the config.yaml to be recreated using these steps:
1. Stop chia
2. Rename the config or move it to a different directory
3. In CLI run chia init
4. Copy any custom settings from the old config to the new (plot drive directories, pool info, trusted node info, etc) - be careful here as yaml is very picky about formatting including those leading spaces
5. Start Chia
```

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
